lib/couchbase-orm/n1ql.rb

```diff
@@ -31,11 +31,11 @@ module ClassMethods
             #    n1ql :by_rating, emit_key: :rating
             #  end
             #
-            #  Post.by_rating.stream do |response|
+            #  Post.by_rating do |response|
             #    # ...
             #  end
             # TODO: add range keys [:startkey, :endkey]
-            def n1ql(name, query: nil, emit_key: [], **options)
+            def n1ql(name, select: nil, emit_key: [], **options)
```

Comment:
why remove callback query args to string select args

Reply:
query_fn is proposed now